data/core/units/monsters/Skeletal_Dragon.cfg

```diff
@@ -45,6 +50,17 @@
         damage=25
         number=2
     [/attack]
+    #Commented out for lack of anim, but potential additional attack idea (on issue post)
+    #[attack]
+    #   name=tail_whip
+    #   description= _"tail_whip"
+    #   type=impact
+    #   range=melee
+    #   [specials]
+    #       {WEAPON_SPECIAL_STUN}
```

Comment:
Just adding this won't work. You didn't add the stun weapon special code to core.

Reply:
If you're not sure how to do this… I've been considering moving the Merman Brawler to core, so I could get that done now and merge it before this PR (assuming it gets approved, of course). Then simply adding the special as done here would be sufficient.

Reply:
> If you're not sure how to do this… I've been considering moving the Merman Brawler to core, so I could get that done now

Is there a separate issue/PR tracking that

Reply:
The closest I'm aware of is #3414.

Reply:
> The closest I'm aware of is #3414.

hm, that one has the Brawler already checked off?

Reply:
That's because there's a commit in that PR that covers the brawler. The ones that have already been merged to master are those that are struck out.